docs/user/integration/dynatrace/README.md

```diff
@@ -162,38 +162,79 @@ To start ingesting custom spans and Istio spans, you must enable the Istio traci
 
 ### Ingest Metrics
 
-To collect custom metrics, you usually use the [Dynatrace annotation approach](https://docs.dynatrace.com/docs/platform-modules/infrastructure-monitoring/container-platform-monitoring/kubernetes-monitoring/monitor-prometheus-metrics), because the Dynatrace OTLP integration is [limited](https://docs.dynatrace.com/docs/extend-dynatrace/opentelemetry/getting-started/metrics/ingest/migration-guide-otlp-exporter#migrate-collector-configuration). As long as your workload is conform to the limitations (not exporting histograms, using delta aggregation temporality), you can use the metric functionality to push OTLP metrics to Dynatrace. In this case, the Prometheus feature of the MetricPipeline cannot be used because it hits the limitations by design.
-
-1. Deploy the MetricPipeline:
-
-    ```bash
-    cat <<EOF | kubectl apply -f -
-    apiVersion: telemetry.kyma-project.io/v1alpha1
-    kind: MetricPipeline
-    metadata:
-        name: dynatrace
-    spec:
-        output:
-            otlp:
-                endpoint:
-                    valueFrom:
-                        secretKeyRef:
-                            name: dynakube
-                            namespace: ${DYNATRACE_NS}
-                            key: apiurl
-                path: v2/otlp/v1/metrics
-                headers:
-                    - name: Authorization
-                      prefix: Api-Token
-                      valueFrom:
-                          secretKeyRef:
-                              name: dynakube
-                              namespace: ${DYNATRACE_NS}
-                              key: dataIngestToken
-                protocol: http
-    EOF
-    ```
-
-1. Start pushing metrics to the metric gateway using [delta aggregation temporality.](https://docs.dynatrace.com/docs/extend-dynatrace/opentelemetry/getting-started/metrics/limitations#aggregation-temporality)
-
-1. To find metrics from your Kyma cluster in the Dynatrace UI, go to **Observe & Explore** > **Metrics**.
+There are several approaches to ingest custom metrics to Dynatrace, each with different benefits and limitations:
+
+- Use a MetricPipeline to push metrics directly.
+   
+> [!NOTE]
+> The Dynatrace OTLP API does [not support](https://docs.dynatrace.com/docs/extend-dynatrace/opentelemetry/getting-started/metrics/ingest/migration-guide-otlp-exporter#migrate-collector-configuration) the full OTLP specification and needs custom transformation. A MetricPipeline does not support these transformation features, so that only metrics can be ingested that don't hit the limitations. At the moment, metrics of type "Histogram" and "Summary" are not supported. Furthermore, "Sum"s must use "delta" aggregation temporality. 
```

Comment:
@NHingerl I made "Histogram" and "Summary" consistently capitalized. Not sure about "delta". They even use it inconsistently in the [OTel docs](https://opentelemetry.io/docs/specs/otel/metrics/data-model/#temporality). What do you think?